docs/content/direct/get-started.md

```diff
@@ -131,9 +131,53 @@ kubectl --context kind-kubeflex wait -n its1-system job.batch/its-with-clusterad
 
 *To learn more about the Core Helm Chart, refer to the [Core Helm Chart documentation](./core-chart.md)*
 
-### Create and register two workload execution cluster(s)
-{% include-markdown "direct/example-wecs.md" heading-offset=2 %}
+### Create and register two workload execution clusters
 
+The following steps show how to create two new `kind` clusters and
+register them with the hub as described in the
+[official open cluster management docs](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/).
+
+Note that `kind` does not support three or more concurrent clusters unless you raise some limits as described in this `kind` "known issue": [Pod errors due to "too many open files"](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files).
+
+1. Execute the following commands to create two kind clusters, named `cluster1` and `cluster2`, and register them with the OCM hub. These clusters will serve as workload clusters. If you have previously executed these commands, you might already have contexts named `cluster1` and `cluster2`. If so, you can remove these contexts using the commands `kubectl config delete-context cluster1` and `kubectl config delete-context cluster2`.
+
+    ```shell
+    : set flags to "" if you have installed KubeStellar on an OpenShift cluster
+    flags="--force-internal-endpoint-lookup"
+    clusters=(cluster1 cluster2);
+    for cluster in "${clusters[@]}"; do
+       kind create cluster --name ${cluster}
+       kubectl config rename-context kind-${cluster} ${cluster}
+       clusteradm --context its1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${cluster}/" | awk '{print $0 " --context '${cluster}' --singleton '${flags}'"}' | sh
+    done
+    ```
+
+    The `clusteradm` command grabs a token from the hub (`its1` context), and constructs the command to apply the new cluster
+    to be registered as a managed cluster on the OCM hub.
```

Comment:
@coderabbitai: your nitpick about these lines looks incorrect to me. I see this text rendered as text rather than code, both when viewed directly on GitHub and when rendered for the website.

<img width="917" alt="image" src="https://github.com/user-attachments/assets/b7d2ce21-afc0-4867-be4f-1aba95f99fdc" />

<img width="946" alt="image" src="https://github.com/user-attachments/assets/52924f00-d25b-46b7-b305-67ae5cdba63d" />